geode-core/src/distributedTest/java/org/apache/geode/internal/cache/DeltaForceSizingFlagDUnitTest.java

```diff
@@ -0,0 +1,460 @@
+      LocalRegion region = (LocalRegion) cache.getRegion(TEST_REGION_NAME);
+      Object value = region.getValueInVM(DeltaForceSizingFlagDUnitTest.DELTA_KEY);
+      switch (expectedType) {
+        case RAW_VALUE:
+          assertFalse("Value was " + value + " type " + value.getClass(),
+              (value instanceof CachedDeserializable));
+          break;
+        case CD_SERIALIZED:
+          assertTrue("Value was " + value + " type " + value.getClass(),
+              value instanceof CachedDeserializable);
+          assertTrue("Value not serialized",
+              ((CachedDeserializable) value).getValue() instanceof byte[]);
+          break;
+        case CD_DESERIALIZED:
+          assertTrue("Value was " + value + " type " + value.getClass(),
+              value instanceof CachedDeserializable);
+          assertFalse("Value was serialized",
+              (((CachedDeserializable) value).getValue() instanceof byte[]));
+          break;
+        case EVICTED:
+          assertNull(value);
+          break;
+      }
+    });
+  }
+
+  private static File[] getMyDiskDirs() {
+    long random = new Random().nextLong();
+    File file = new File(Long.toString(random));
+    assertTrue(file.mkdirs());
+    return new File[] {file};
+  }
+
+  private void createPR(MemberVM memberVM) {
+    memberVM.invoke("Create partitioned region", () -> {
+      Cache cache = ClusterStartupRule.getCache();
+      assertNotNull(cache);
+
+      RegionFactory<Integer, TestDelta> regionFactory = cache.createRegionFactory();
+
+      regionFactory.setDiskSynchronous(true);
+      PartitionAttributesFactory<Integer, TestDelta> paf =
+          new PartitionAttributesFactory<>();
+      paf.setRedundantCopies(1);
+      PartitionAttributes<Integer, TestDelta> prAttr = paf.create();
+      regionFactory.setPartitionAttributes(prAttr);
+      regionFactory.setDataPolicy(DataPolicy.PARTITION);
+      regionFactory.setSubscriptionAttributes(new SubscriptionAttributes(InterestPolicy.ALL));
+      regionFactory.create(TEST_REGION_NAME);
+    });
+  }
+
+  private static class TestObjectSizer implements ObjectSizer {
+    private final AtomicInteger invocations = new AtomicInteger();
+
+    @Override
+    public int sizeof(Object o) {
+      logger.debug("TestObjectSizer invoked"/* , new Exception("stack trace") */);
+      if (o instanceof TestObject) {
+        invocations.incrementAndGet();
+        return ((TestObject) o).sizeForSizer;
+      }
+      if (o instanceof TestDelta) {
+        invocations.incrementAndGet();
+        return ((TestDelta) o).info.length();
+      }
+      if (o instanceof Integer) {
+        return 0;
+      }
+      if (o instanceof TestKey) {
+        invocations.incrementAndGet();
+        return ((TestKey) o).value.length();
+      }
+      throw new RuntimeException("Unexpected type to be sized " + o.getClass() + ", object=" + o);
+    }
+  }
+
+  private static class TestKey implements DataSerializable {
```

Comment:
This class is no longer instantiated anywhere in the test, so can be removed.

Reply:
This change was marked as resolved, but the unused internal class is still present.